src/Pages/WishList/PaymentEmail.js

```diff
@@ -0,0 +1,96 @@
+import React from "react";
+import EmailGoodsList from "../../Components/EmailGoodsList";
+import "./PaymentEmail.scss";
+import "../../Styles/common.scss";
+
+class PaymentEmail extends React.Component {
+  constructor() {
+    super();
+    this.state = {
+      Goods: [],
+    };
+  }
+
+  componentDidMount = () => {
+    fetch("/data/mockData/Goods.json", {})
```

Comment:
```suggestion
    fetch("/data/mockData/Goods.json")
```
get 요청일 경우 두번째 인자에 아무것도 전달하지 않아도 됩니다.

Reply:
인자 제거 후 간략화 완료하였습니다